native/pipewire-screenaudio/main.cpp

```diff
@@ -9,6 +9,13 @@ std::optional<pipewire::port> virt_fl, virt_fr;
 std::map<std::uint32_t, pipewire::node> nodes;
 std::map<std::uint32_t, pipewire::link> links;
 
+std::vector<std::string> EXCLUDE_TARGETS{"AudioCallbackDriver"};
```

Comment:
We should change this to exclude the process ID of the call node, because you will not be able to stream a call while in another call.